src/components/Operators.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line react/prop-types
```

Comment:
- Kindly fix this linter error instead of disabling it. add the propType for the click. please, let's avoid disabling the linter.
- Another thing is, according to the previous reviewer, you should open different pull requests for different project milestones. Now, you've worked on the API inside the Hooks milestone. please, let's be mindful of this for subsequent projects.

Reply:
noted, issue fixed